### PR TITLE
Add Image List page

### LINKED ISSUE #26: Create Image List component page

### PR DESCRIPTION
**Pull request by kfranqueiro**
Resolves #26.

### CODE REVIEW

File: src/ImageListPage.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    standardImageListItems.push({
+      src: `${imagePath}/photos/3x2/${i}.jpg`,
+      label: 'Text label'
+    });
+  }
+
+  const masonryImageListItems = [
+    {
+      src: `${imagePath}/photos/3x2/16.jpg`,
+      label: 'Text label',
```
Comment (moog16) [inline src/ImageListPage.js]: can we get rid of the label property here? Its the same in every object, so just hard code it. Then all these 2 arrays of objects can just be 2 arrays of strings. 

Also seems like we could inline `${imagePath}/photos` in the images. 
```
<ImageListItem src={`${imagePath}/photos/${src}`} ... />
```

The arrays would then look like:

```
[ '3x2/16.jpg', '2x3/1.jpg, ... ]
```

File: src/ImageListPage.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+function ImageListPage() {
+  return (
+    <div>
+      <HeaderBar />
+      <ComponentPage
+        hero={<ImageListHero />}
+        title='Image List'
+        description='Image lists display a collection of images in an organized grid.'
+        designLink='https://material.io/go/design-image-list'
```
Comment (moog16) [inline src/ImageListPage.js]: FYI this goes to a COMING SOON! page. Just wanna make sure that is intendend.

Comment (lynnjepsen) [reply src/ImageListPage.js]: ya thats right

File: src/ImageListPage.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      label: 'Text label',
+    },
+  ];
+
+  return (
+    <div>
+      <h3 className='mdc-typography--subheading2'>Standard Image List with Text Protection</h3>
+      <ImageList className='standard-image-list mdc-image-list--with-text-protection' items={standardImageListItems}
+        includeAspectContainer />
+      <h3 className='mdc-typography--subheading2'>Masonry Image List with Text Labels Underneath</h3>
```
Comment (lynnjepsen) [inline src/ImageListPage.js]: nit: drop the "with Text Labels Underneath"...this is the default option.

**Pull request merged**